src/containers/SocialPage/IntroBlock.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { Link } from 'react-router'
+import instagram_logo from '../../assets/instagram_logo.png'
+import facebook_logo from '../../assets/facebook_logo.png'
+import snapchat_logo from '../../assets/snapchat_logo.png'
+import twitter_logo from '../../assets/twitter_logo.png'
+
+class IntroBlock extends React.Component {
+
+  static propTypes = {
+    data: React.PropTypes.object,
+  }
+  
+  render () {
+
+    return (
+      <div className="db dt-ns mw9 center">
+        <div className="fl-m fl-l w-50-m w-50-l">
+          <div className="pa4 pa4-m">
+          <h2 className="f2">Facebook</h2>
+            <img className="logo" src={facebook_logo}/>
```

Comment:
Suggestion: Have you thought of wrapping the image with a <Link>? There is a link you can use to take our users to the relevant page.... maybe we can implement a Facebook "badge" or whatever they call them? But looking good!